src/sql/workbench/api/browser/mainThreadWorkspace.ts

```diff
@@ -18,24 +19,27 @@ export class MainThreadWorkspace extends Disposable implements MainThreadWorkspa
 		extHostContext: IExtHostContext,
 		@IWorkspaceEditingService private workspaceEditingService: IWorkspaceEditingService,
 		@IWorkbenchEnvironmentService protected readonly environmentService: IWorkbenchEnvironmentService,
+		@IHostService private readonly hostService: IHostService
 	) {
 		super();
 	}
 
-	$createAndEnterWorkspace(folder: URI, workspaceFile?: URI): Promise<void> {
+	async $createAndEnterWorkspace(folder: URI, workspaceFile?: URI): Promise<void> {
 		folder = URI.revive(folder);
 		workspaceFile = URI.revive(workspaceFile);
-		return this.workspaceEditingService.createAndEnterWorkspace([{ uri: folder }], workspaceFile);
+		await this.workspaceEditingService.createAndEnterWorkspace([{ uri: folder }], workspaceFile);
+		this.hostService.reload();
```

Comment:
does it affect any tests

Reply:
it doesn't affect any of the data workspace or sql database projects tests since these are stubbed